buku.py

```diff
@@ -1658,44 +1728,12 @@ def importdb(self, filepath, tacit=False):
             else:
                 resp = 'y'
 
-            for tag in soup.findAll('a'):
-                # Extract comment from <dd> tag
-                try:
-                    if (is_nongeneric_url(tag['href'])):
-                        continue
-                except KeyError as e:
-                    continue
-
-                desc = None
-                comment_tag = tag.findNextSibling('dd')
-
-                if comment_tag:
-                    desc = comment_tag.find(text=True, recursive=False)
-
-                # add parent folder as tag
-                if resp == 'y':
-                    # could be its folder or not
-                    possible_folder = tag.find_previous('h3')
-                    # get list of tags within that folder
-                    tag_list = tag.parent.parent.find_parent('dl')
-
-                    if ((possible_folder) and
-                            possible_folder.parent in list(tag_list.parents)):
-                        # then it's the folder of this bookmark
-                        if tag.has_attr('tags'):
-                            tag['tags'] += (DELIM + possible_folder.text)
-                        else:
-                            tag['tags'] = possible_folder.text
-
-                # add unique tag if opted
-                if newtag:
-                    if tag.has_attr('tags'):
-                        tag['tags'] += (DELIM + newtag)
-                    else:
-                        tag['tags'] = newtag
-
-                self.add_rec(tag['href'], tag.string, parse_tags([tag['tags']])
-                             if tag.has_attr('tags') else None, desc, 0, True)
+            add_parent_folder_as_tag = resp == 'y'
```

Comment:
Can you please change this to:

    add_parent_folder_as_tag = (resp == 'y')

?